drivers/net/wireless/mediatek/mt76/mt76_connac_mcu.c

```diff
@@ -218,7 +218,7 @@ void mt76_connac_mcu_beacon_loss_iter(void *priv, u8 *mac,
 	if (!(vif->driver_flags & IEEE80211_VIF_BEACON_FILTER))
 		return;
 
-	ieee80211_beacon_loss(vif);
+	ieee80211_connection_loss(vif);
```

Comment:
@objelf is it true for mt7663 fw as well?

Reply:
The change should be dropped. let's focus on mt7921 first.